gremlin-scala/src/main/scala/gremlin/scala/ScalaVertex.scala

```diff
@@ -50,35 +54,48 @@ case class ScalaVertex(vertex: Vertex) extends ScalaElement[Vertex] {
 
   def bothE(labels: String*) = start().bothE(labels: _*)
 
-  def addEdge(
-    label: String,
-    inVertex: ScalaVertex,
-    properties: Map[String, Any] = Map.empty
-  ): ScalaEdge = {
-    val params = properties.toSeq.flatMap(pair ⇒ Seq(pair._1, pair._2.asInstanceOf[AnyRef]))
+  def addEdge(label: String,
+              inVertex: ScalaVertex,
+              properties: Map[String, Any] = Map.empty): ScalaEdge = {
+    val params = properties.toSeq.flatMap(pair => Seq(pair._1, pair._2.asInstanceOf[AnyRef]))
     vertex.addEdge(label, inVertex.vertex, params: _*)
   }
 
-  def addEdge[P <: Product: Marshallable](inVertex: ScalaVertex, cc: P): ScalaEdge = {
+  def addEdge[P <: Product : Marshallable](inVertex: ScalaVertex, cc: P): ScalaEdge = {
     val (id, label, properties) = implicitly[Marshallable[P]].fromCC(cc)
     val idParam = id.toSeq flatMap (List(T.id, _))
-    val params = properties.toSeq.flatMap(pair ⇒ Seq(pair._1, pair._2.asInstanceOf[AnyRef]))
+    val params = properties.toSeq.flatMap(pair => Seq(pair._1, pair._2.asInstanceOf[AnyRef]))
```

Comment:
Sorry, that's a mistake.
The thing is that I can't put back the special char by copy and pasting it :/